core/market/decentralized/src/db/model/events.rs

```diff
@@ -48,15 +49,15 @@ pub struct MarketEvent {
     pub event_type: EventType,
     /// It can be Proposal, Agreement or structure,
     /// that will represent PropertyQuery.
-    pub artifact_id: String,
+    pub artifact_id: ProposalId,
 }
 
 #[derive(Clone, Debug, Insertable)]
 #[table_name = "market_event"]
 pub struct NewMarketEvent {
     pub subscription_id: SubscriptionId,
     pub event_type: EventType,
-    pub artifact_id: String, // TODO: typed
+    pub artifact_id: ProposalId, // TODO: typed
```

Comment:
What with PropertyQuery?

Reply:
it is not in scope of MVP. So let's not bother now

Reply:
That's why I don't think you should change it